src/components/AttentionBox/__tests__/__snapshots__/attentionBox-snapshot-tests.jest.js.snap

```diff
@@ -238,14 +238,12 @@ exports[`AttentionBox renders correctly renders correctly with onClose 1`] = `
     Text
   </div>
   <svg
-    aria-hidden={false}
+    aria-hidden={true}
```

Comment:
change it here to be clickable or merge the attention box pr before (because the close button should be clickable)

Reply:
Attention box PR merged here